src/goto-programs/goto_program.h

```diff
@@ -588,28 +588,6 @@ class goto_programt
     return t;
   }
 
-  optionalt<const_targett> get_target(const unsigned location_number) const
```

Comment:
I take it this was only used by goto-harness.

Reply:
It was used by the option `--initial-location` to find the instruction with this `location_number`. Now that we have two ways for the user to specify the entry point I generalised the search for the right instruction and thus `get_target` was no longer needed.

Reply:
OK; sounds good.